### ISSUE TITLE
Davis and YouTube datasets are multi-targeted, right?

### ISSUE THREAD

**Issue by longmalongma** [hkchengrex/STCN#57]
Davis and YouTube datasets are multi-target, right? Or is there a single target? Self. single_object is also important?

![image](https://user-images.githubusercontent.com/40291722/133114904-44ee7207-6d5b-43e4-9e0d-56a95fc62182.png)

**Comment by hkchengrex** [comment]
That flag is used for static image pretraining only.

**Comment by longmalongma** [comment]
> That flag is used for static image pretraining only.

@hkchengrex You mean there is no single target in YouTube and davis2017 datasets?However, I saw the mask of single target in the ground Truth of YouTube and Davis2017 data set. Now the test accuracy rate is close to 0 after I change the code. I suspect that only the code of multiple targets has been changed, but the code of single target has not been changed.(No pre-training)

**Comment by hkchengrex** [comment]
The `single_object` flag is set here: https://github.com/hkchengrex/STCN/blob/e03c1585fd852624204b5c4e8d8e828ddb2d406f/util/hyper_para.py#L59

So it should not enter the single object path if no pretraining is done.
